### DISCUSSION TITLE
Is there a way to download the chunks files?

### DISCUSSION THREAD

**Discussion by cccandice404** [infiniflow/ragflow#2717] [Q&A]
As I want to further edit the chunks, can I download the chunks files to my host file system, assuming I'm running this on Docker? I tried to trace the code as it stored at "self.post('/doc/chunk/set')" but then still find it difficult to further trace. Thank you so much!!

**Comment by KevinHuSh** [comment]
Checkout [this](https://github.com/infiniflow/ragflow/blob/main/sdk/python/test/t_document.py#L229), which is about usage of RAGFlow SDK.
The official documents are coming soon.
